flutter_vlc_player/example/lib/single_tab.dart

```diff
@@ -127,7 +127,26 @@ class _SingleTabState extends State<SingleTab> {
       children: [
         Container(
           height: 400,
-          child: VlcPlayerWithControls(key: _key, controller: _controller),
+          child: VlcPlayerWithControls(
+            key: _key,
+            controller: _controller,
+            onStopRecording: (recordPath) {
+              listVideos.add(VideoData(
+                name: 'Recorded Video',
+                path: recordPath,
+                type: VideoType.recorded,
+              ));
```

Comment:
Please move this inside the `setState` callback -- this way it will be clearer which piece of state is changing.

Reference: https://api.flutter.dev/flutter/widgets/State/setState.html
> Whenever you change the internal state of a State object, make the change in a function that you pass to setState: